third_party/suitesparse.BUILD

```diff
@@ -0,0 +1,600 @@
+"""
+suitesparse@7.1.0
+"""
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "suitesparseconfig",
+    srcs = [
+        "SuiteSparse_config/SuiteSparse_config.c",
+    ],
+    hdrs = [
+        "SuiteSparse_config/SuiteSparse_config.h",
+    ],
+    includes = [
+        "SuiteSparse_config/",
+    ],
+    strip_include_prefix = "SuiteSparse_config",
```

Comment:
For third party code we should try to avoid using `strip_include_prefix` which causes the include path to be propagated with `-I` - The `includes` statements above should be sufficient.

Reply:
Great -- while developing, I thought I needed this for e.g. `cholmod` to correctly get headers from `amd` etc., but it looks like you're right.